re-former/config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+	resources :users
```

Comment:
you should cover only `:new` ,`:create`,`:edit`,`:update`

Reply:
Modified as advised